pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/QuestionDao.java

```diff
@@ -1545,7 +1546,10 @@ default void insertQuestion(long activityId, PicklistQuestionDef picklist, long
 
     default void insertQuestion(long activityId, CompositeQuestionDef compositeQuestion, long revisionId) {
         if (!compositeQuestion.isAcceptable()) {
-            throw new DaoException("Composites only support DATE, PICKLIST, TEXT, NUMERIC and DECIMAL child questions");
+            throw new DaoException("Composite questions are only support following types for children: "
```

Comment:
Need to remove "are" from the sentence
Composite questions only support following types for children:

Reply:
Rest all looks good to me

Reply:
Thank you :)
Removed.